libnet/Makefile.am.common

```diff
@@ -5,7 +5,6 @@
 # All rights reserved.
```

Comment:
I see. Inappropriate indeed. I already removed the whole file with the my last commit to my fork. Now I can safely get rid of this custom distclean rule as well.